src/navigation/MainTabNavigator.tsx

```diff
@@ -9,101 +13,154 @@ import HomeStack from "./HomeStack"
 import SymptomHistoryStack from "./SymptomHistoryStack"
 import SettingsStack from "./SettingsStack"
 import { useConfigurationContext } from "../ConfigurationContext"
-
 import { Stacks } from "./index"
 import { TabBarIcons } from "../assets/svgs/TabBarNav"
-import { Colors } from "../styles"
+import { Text } from "../components"
 
-const Tab = createBottomTabNavigator()
+import { Colors, Layout, Outlines, Spacing, Typography } from "../styles"
 
 const MainTabNavigator: FunctionComponent = () => {
   const { t } = useTranslation()
   const insets = useSafeAreaInsets()
   const { displaySymptomHistory } = useConfigurationContext()
 
-  interface TabIconProps extends TabBarIconProps {
-    icon: string
+  interface TabBarIconProps {
+    focused: boolean
+  }
+
+  const HomeIcon: FunctionComponent<TabBarIconProps> = ({ focused }) => {
+    return <TabIcon icon={TabBarIcons.House} focused={focused} />
   }
 
-  const TabIcon: FunctionComponent<TabIconProps> = ({
+  const ExposureHistoryIcon: FunctionComponent<TabBarIconProps> = ({
     focused,
-    size,
-    icon,
   }) => {
+    const tabIcon = <TabIcon icon={TabBarIcons.Exposure} focused={focused} />
+    return tabIcon
+  }
+
+  const HeartbeatIcon: FunctionComponent<TabBarIconProps> = ({ focused }) => {
+    const tabIcon = <TabIcon icon={TabBarIcons.Heartbeat} focused={focused} />
+    return tabIcon
+  }
+
+  const SettingsIcon: FunctionComponent<TabBarIconProps> = ({ focused }) => {
+    const tabIcon = <TabIcon icon={TabBarIcons.Gear} focused={focused} />
+    return tabIcon
+  }
+
+  interface TabIconProps extends TabBarIconProps {
+    icon: string
+  }
+
+  const TabIcon: FunctionComponent<TabIconProps> = ({ focused, icon }) => {
+    const iconSize = 22
+
     return (
       <SvgXml
         xml={icon}
         fill={focused ? Colors.primary.shade100 : Colors.neutral.shade50}
-        width={size}
-        height={size}
+        width={iconSize}
+        height={iconSize}
       />
     )
   }
 
-  interface TabBarIconProps {
-    focused: boolean
-    size: number
-  }
-
-  const HomeIcon: FunctionComponent<TabBarIconProps> = ({ focused, size }) => {
-    return <TabIcon icon={TabBarIcons.House} focused={focused} size={size} />
+  type Tab_ = "home" | "exposureHistory" | "settings" | "symptomHistory"
+  const tabs: Tab_[] = ["home", "exposureHistory", "settings"]
+  if (displaySymptomHistory) {
+    tabs.push("symptomHistory")
```

Comment:
wouldnt this put symptom history on end? do we not want settings to stay on the end?